pegasus/helpers/section_api_helpers.rb

```diff
@@ -393,7 +393,7 @@ def self.valid_courses
       select(:id, :name).
       all.
       # Only return courses we've whitelisted in ScriptConstants
-      select {|course| ScriptConstants.script_in_category?(:full_course, course_assignment_family(course))}.
+      select {|course| ScriptConstants.script_in_category?(:full_course, course[:name])}.
```

Comment:
Unrelated to this change, but it's unfortunate that we're passing a course name into a method called `script_in_category?`.

Reply:
Agreed 😞 It's also sad that this is in module called ScriptConstants. Our term for "course or script" is "assignment", so perhaps we could start using that term here in the future.